### PR TITLE
Add a load of tests

### PR DESCRIPTION
**Pull request by martindurant**
Pushes coverage over 90%

Anyone know why the coveralls upload isn't working?

### CODE REVIEW

File: intake/cli/client/subcommands/drivers.py [hunk 1]
```diff
@@ -67,15 +72,24 @@ def _list(self, args):
             fmt = '{name:<30}{cls.__module__}.{cls.__name__}'
         drivers_by_name = autodiscover()   # dict mapping name to driver
         all_drivers = autodiscover_all()  # listof (name, driver)
-        print("Enabled:", file=sys.stderr)
+        direct = {k: v for k, v in intake.registry.items()
+                  if k not in all_drivers and k not in drivers_by_name}
+
+        print("Direct:", file=sys.stderr)
```
Comment (martindurant) [inline intake/cli/client/subcommands/drivers.py]: @danielballan - these are all the drivers that simply assign themselves to the registry. Should it be possibel to disable them? I'd say yes; perhaps that means that Intake itself should declare its own entrypoints.
Also, I do you think that "enable" should be able to use a name alone, if that name is in the current disable list? Obviously, that only works for unique names, so perhaps not.

Comment (danielballan) [reply intake/cli/client/subcommands/drivers.py]: >  I'd say yes; perhaps that means that Intake itself should declare its own entrypoints.
I agree. That's what `nbconvert` does.

> Also, I do you think that "enable" should be able to use a name alone, if that name is in the current disable list? Obviously, that only works for unique names, so perhaps not.

Yeah, I lean towards no for that reason. Let's see if people ask for it much.

Comment (danielballan) [reply intake/cli/client/subcommands/drivers.py]: > I'd say yes; perhaps that means that Intake itself should declare its own entrypoints.

I agree. That's what nbconvert does.

> Also, I do you think that "enable" should be able to use a name alone, if that name is in the current disable list? Obviously, that only works for unique names, so perhaps not.

Yeah, I lean towards no for that reason. Let's see if people ask for it much.

Comment (martindurant) [reply intake/cli/client/subcommands/drivers.py]: OK, let me try to add entrypoints to intake itself, and see if those "direct" entries all go away.

**Pull request merged**